src/cli/local-commands/build.command.ts

```diff
@@ -19,9 +22,33 @@ export async function handler() {
 
   await stage('Building...', async () => {
     await runPluginLoaderHook(plugins, (loader) => loader.applicationWillBuild())
+    const compiler = webpack(getWebpackConfig({
+      pages: getEntrypointFiles(),
+      plugins,
+      devServer: false
+    }))
+
+    await runCompiler(compiler)
   })
 
+  function getEntrypointFiles() {
+    const filePattern = path.join('src', 'pages', '**', '*.@(t|j)s?(x)')
```

Comment:
Didn't you have to take this path.join out to solve the issue with Windows path - is this re-instating it?

Reply:
👍 There's also some duplication between the build and run commands which should probably be extracted